articles/batch/batch-upgrade-policy.md

```diff
@@ -0,0 +1,179 @@
+
+## How does Auto OS Upgrade work?
+
+When upgrading images, VMs in Azure Batch Pool will follow the same work flow as VirtualMachineScaleSets. An upgrade works by **replacing the OS disk of a VM with a new disk created using the latest image version**. Any configured extensions and custom data scripts are run on the OS disk, while data disks are retained. 
+
+By default, upgrades take place in batches, with no more than 20% of the scale set upgrading at any time. The upgrades take place during off-peak hours of a region by default, generally between 10 PM to 6 AM local time.
+
+The eligibility for image upgrades in a region with a scale set can occur either through the availablity-first process for platform images or custom images:
+
+* For platform images, the upgrade process follows a monthly schedule to roll out supported OS platform image upgrades.
+* For custom images, the upgrade takes place when a new image is published and replicated to the specific region of that scale set.
+
+The image upgrade is then applied to an individual scale set in a batched manner as follows:
+
+1. Before commencing the upgrade, the orchestrator ensures that no more than 20% of instances in the entire scale set are in an unhealthy state, regardless of the reason for their health status.
+2. The upgrade orchestrator identifies a batch of VM instances to be upgraded. Each batch contains a maximum of 20% of the total instance count, with a minimum batch size of one virtual machine. There is no minimum scale set size requirement, and scale sets with 5 or fewer instances will have 1 VM per upgrade batch as the minimum batch size.
+3. The OS disk of each VM within the selected upgrade batch is replaced with a new OS disk created from the latest image version. All specified extensions and configurations in the scale set model are applied to the upgraded instance.
+4. The upgrade process includes a waiting period of up to 5 minutes for an instance to regain a healthy state before proceeding to upgrade the next batch. If an instance fails to recover its health within this 5-minute window after an upgrade, the previous OS disk for that instance is automatically restored by default.
+5. The upgrade orchestrator also monitors the percentage of instances that become unhealthy after an upgrade. If more than 20% of the upgraded instances become unhealthy during the upgrade process, the upgrade will be halted.
+6. This entire process continues until all instances within the scale set have undergone the upgrade, ensuring that all instances are running the latest OS image while minimizing disruptions and ensuring overall system health.
+
+## Supported OS images
+Only certain OS platform images are currently supported. Custom images are supported if the pool uses custom images through [Azure Compute Gallery](../virtual-machines/shared-image-galleries.md).
+
+The following platform SKUs are currently supported (and more are added periodically):
```

Comment:
Since CRP maintains this lit, we don't need to list them here. Provide link is good enough, otherwise we will have to keep monitoring it and keep it in sync with CRP